pkg/settlement/pseudosettle/pseudosettle.go

```diff
@@ -272,7 +272,7 @@ func (s *Service) Pay(ctx context.Context, peer swarm.Address, amount, checkAllo
 	}
 
 	currentTime := s.timeNow().Unix()
-	if currentTime == lastTime.CheckTimestamp {
+	if currentTime == lastTime.CheckTimestamp || currentTime == lastTime.Timestamp {
```

Comment:
I know that previously this check was here but @ralph-pichler replaced it with the `CheckTimestamp` field stating that checking against `Timestamp` was incorrect. @ralph-pichler can you elaborate?

Reply:
( comment from review: peer timestamp can be preemptively checked in accounting as well to inhibit ErrSettlementTooSoon)